Cesium.Preprocessor/BinaryExpression.cs

```diff
@@ -2,23 +2,20 @@
 
 namespace Cesium.Preprocessor;
 
-internal sealed class BinaryExpression : IPreprocessorExpression
+public class BinaryExpression(
```

Comment:
I'd prefer these classes to stay `internal`, but add `InternalsVisibleTo` to this assembly for the tests exclusively.